committeeoversightapp/models.py

```diff
@@ -189,7 +189,7 @@ def is_current(self):
     def percent_passed(self):
         days_in_session = 668
         days_passed = (date.today() - self.start_date).days
-        percent_passed = round(days_passed / days_in_session * 100, 2)
+        percent_passed = int(round(days_passed / days_in_session * 100))
```

Comment:
Do you need both `int()` _and_ `round()`?

Reply:
Nope you're right! `round()` with no second argument returns an integer.